templates/Managerpage.html

```diff
@@ -12,18 +12,28 @@
 <form action="/managerSearch" method="post">
 <center><b>Generate Product orders</b></center>
 <p>

```

Comment:
What would happen if you took this `<p>` tag out? or at least close it? Would that make the button happy again?